src/screens/ComponentsScreen.tsx

```diff
@@ -46,10 +47,37 @@ export const ComponentsScreen = (): JSX.Element => {
         ))}
         <Button mb={2} isLoading variant="link" />
         <Icon name="account-box-fill" size={24} color={'amber.600'} />
-        <Button mb={4} isDisabled>
+        <Button my={4} isDisabled>
           {t('components_screen.button_variants.disabled')}
         </Button>
+        <Divider my={4} />
```

Comment:
Please use `.map` function over the array with loaders types to avoid unnecessary repetitions